.golangci.yml

```diff
@@ -0,0 +1,38 @@
+linters:
```

Comment:
Is this `treefmt`'s list? If this is someone else's list we're aiming to keep in sync with, let's add a link?